andela/AndelaEatsUI
# #161337268 Connect meal item get and post requests to API

#### What does this PR do?
- Connect meal item get and post requests to API

#### Description of Task to be completed?
- Fetch meal items from real api
- Upload new meal item selected image to cloudinary
- Post new meal items to real api

#### How should this be manually tested?
- Start and login to the app after cloning and checking out to this branch
- Navigate to the admin dashboard from the user drop-down
- Switch to the `Meal Items` tab from the side nav
- Ensure meal items are loaded to the screen
- Click on `Add meal item` button and provide meal details
- Ensure to see the main screen updated with new meal items and that it is still seen on refresh.

#### What are the relevant pivotal tracker stories?
- [#161337268](https://www.pivotaltracker.com/story/show/161337268)

#### Screenshots (If Applicable)
<img width="1440" alt="screen shot 2018-10-22 at 4 16 32 pm" src="https://user-images.githubusercontent.com/32123313/47301033-de7a1480-d615-11e8-905a-ba0226c0e2bd.png">
<img width="1440" alt="screen shot 2018-10-22 at 4 16 37 pm" src="https://user-images.githubusercontent.com/32123313/47301034-de7a1480-d615-11e8-81e6-f1510fd7602c.png">
